docs/tutorials/02-registries-repositories.md

```diff
@@ -0,0 +1,133 @@
+# Buildah Tutorial 2
+## Using Buildah with container registries
+
+The purpose of this tutorial is to demonstrate how Buildah can be used to move OCI compliant images in and out of private or public registries. 
+
+In the [first tutorial](https://github.com/projectatomic/buildah/blob/master/docs/tutorials/01-intro.md) we built an image from scratch that we called `fedora-bashecho` and we pushed it to a local docker repository using the `docker-daemon` protocol. We are going to use the same image to push to a private docker registry.
+
+First we must pull down a registry using:
+
+    # registry=$(buildah from registry)
```

Comment:
I'm on the fence.  Should you explain that this is saving the container's id  in a bash variable for easy manipulation later?

Reply:
I had done that in the first tutorial but I could do it again here if you want.

Reply:
Explained.